### ISSUE TITLE
[wish] make use of the eal files to use the EAX/EFX effects OpenAL provides

### ISSUE THREAD

**Issue by senneschall** [OldUnreal/UnrealTournamentPatches#1494]
For UT 436 there were those patches made by Creative to utilize EAX on most default maps. [OldUnreal archived those patches ](https://oldunreal.com/downloads/unrealtournament/other-patches/).
The eaxutsys.exe contains the Crative EaxMan.dll and apatched Galaxy.dll, the other eaxut* files contain eal files for 31 maps.

As far as I understood, the Galaxy.dll is patched in a way so it can use EAX hardware effects. In other words, you'd need a Creative soundcard and use ALchemy to make it work. But when it works, it sounds fantastic, see [example #1](https://drive.google.com/file/d/0B_YWyIfXsVfYdy1wYUNJTHEtNWc/view?resourcekey=0-xUfpGV1Fa_4aUS0b1se0-A) & [example #2](https://drive.google.com/file/d/0B_YWyIfXsVfYNzNvTWVfaU9OeTQ/view?resourcekey=0-98b_81oX9gvbQdili64R1A). Since you need that specific hardware, need Windows and the old Galaxy.dll for 436, it's quite useless today.

The OpenAL driver with v469 would support all those effects as well. But as far as i'm aware there's no way to use the information stored in the *.eal files to use them. You'd need to modify the maps and include the EFX zones into the map, but modified maps lose online support.

So my question is: is it possible to include support for those *.eal files in the 469 OpenAL driver so we can enjoy those sound effects?

**Comment by brdempsey69** [comment]
Yes, I do 2nd this request. OpenAl is pretty good and has great potential, but there is still that issue with all the background noise in CTF-LavaGiant no matter where you are in the map, whereas with a Creative sound card using Alchemy and the Galaxy audio driver,  you don't hear any of that noise.

**Comment by DDRRE** [comment]
+1 on this.

I tried using the v436 Galaxy.dll from the EAX patch (and getting UT to apply EAX using DSOAL) but that obviously failed and the game crashed. Logs and process trace didn't show anything valuable.

The regular Galaxy.dll doesn't seem to utilize the .eal files.

**Comment by senneschall** [comment]
I've tried to collect as much info as possible and see through a sample implementation. And I think, it is doable to make those eal presets work in modern UT patches, even tough there is still some decoding work to do. I've written down the details here: https://github.com/senneschall/EAXManagerEAL
